jinja.md

```diff
@@ -0,0 +1,95 @@
+---
+title: jinja
```

Comment:
While the package is named [jinja2](https://pypi.org/project/Jinja2/), the [docs](https://jinja.palletsprojects.com/en/3.0.x/) only mention "jinja", not "jinja2", and jinja2 [v3.0](https://jinja.palletsprojects.com/en/3.0.x/changes/) would sound weird. [Jinja 1.x](https://pypi.org/project/Jinja/) has seen no release since 2007 so I guess nobody uses it anymore. What do you think?

Reply:
Interesting, we can probably do `jinja2` with an alias (redirect) for `jinja`. Similar to `devhints.io/sh` and `devhints.io/bash`

Reply:
@rstacruz done, I believe